src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
```

Comment:
Yes! This was an exciting branch for us :)
I agree with your point, and I did make some (okay, quite a few) changes yesterday to Cromwell, to remove some of the backend references. I was hoping to push the branch to the repo so that you guys can see what we can be expecting in the future and point to changes in the design once we're back from our vacations. The changes are by no means anything final, and will change a lot once we agree on a design, but it's just to give an idea to everyone. Currently, it's based out of the master branch, so later we will merge it to the latest master of that time (likely after CallCaching has been merged and a new cromwell 0.15 is released)